pkg/label/label.go

```diff
@@ -0,0 +1,17 @@
+package label
+
+const (
+	App = "app"
+)
+
```

Comment:
Why separate blocks of constants?

Reply:
No clue. I just copied this from `aws-operator` as that is SOTA. It seems like logical grouping.